web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -427,4 +428,14 @@ public static function ensureUrlIsString(string|GeneratedUrl $url): string {
     return $url;
   }
 
+  /**
+   * Get the instant loan configuration.
+   *
+   * @return mixed[]
+   *   The instant loan configuration.
+   */
+  public static function getInstantLoanConfig() {
```

Comment:
Please add the return type to the method instead of in the Docblock.

Reply:
👍